manifests/07-downloads-deployment.yaml

```diff
@@ -116,8 +116,15 @@ spec:
           # IPv6 should handle IPv4 passively so long as it is not bound to a
           # specific address or set to IPv6_ONLY
           # https://stackoverflow.com/questions/25817848/python-3-does-http-server-support-ipv6
-          addr = ('::', 8080)
-          sock = socket.socket(socket.AF_INET6, socket.SOCK_STREAM)
+          try:
+              addr = ('::', 8080)
+              sock = socket.socket(socket.AF_INET6, socket.SOCK_STREAM)
+          except socket.error as err:
+              # errno.EAFNOSUPPORT is "socket.error: [Errno 97] Address family not supported by protocol"
+              # When IPv6 is disabled, socket will bind using IPv4.
+              if err.errno == errno.EAFNOSUPPORT:
+                  addr = ('', 8080)
+                  sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
I think you need an:

```python
else:
    raise
```

here to handle the "not `EAFNOSUPPORT`" case.

Reply:
@wking Thank you for your review, I modified as you mentioned.